src/Superstructure/State/src/ESMF_StateAPI.cppF90

```diff
@@ -1970,6 +1971,8 @@ type(ESMF_KeywordEnforcer), optional:: keywordEnforcer ! must use keywords below
          msgbuf = "Import State"
        case (ESMF_STATEINTENT_EXPORT%state)
          msgbuf = "Export State"
+       case (ESMF_STATEINTENT_INTERNAL)
```

Comment:
It looks like this should be `case (ESMF_STATEINTENT_INTERNAL%state)`.

Reply:
Yup.  Thought I could just double check my work and avoid a test compilation in my rush before leaving on travel.   Am back in town and will fix and resubmit tomorrow (I hope).